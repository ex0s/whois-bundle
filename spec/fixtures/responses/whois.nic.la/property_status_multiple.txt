This whois service is provided by CentralNic Ltd and only contains
information pertaining to Internet domain names we have registered for
our customers. By using this service you are agreeing (1) not to use any
information presented here for any purpose other than determining
ownership of domain names, (2) not to store or reproduce this data in 
any way, (3) not to use any high-volume, automated, electronic processes
to obtain data from this service. Abuse of this service is monitored and
actions in contravention of these terms will result in being permanently
blacklisted. All data is (c) CentralNic Ltd https://www.centralnic.com/

Domain ID:CNIC-DO469366
Domain Name:PLASTICSURGERY.LA
Created On:02-Feb-2004 01:00:00 UTC
Last Updated On:31-Mar-2010 12:43:59 UTC
Expiration Date:02-Feb-2017 23:59:59 UTC
Status:TRANSFER PROHIBITED
Status:RENEW PERIOD
Registrant ID:P-1901109
Registrant Name:Jay Granzow
Registrant Organization:Jay Granzow
Registrant Street1:22930 Crenshaw Blvd. Suite D
Registrant City:Torrance
Registrant State/Province:CA
Registrant Postal Code:90505
Registrant Country:US
Registrant Phone:+1.3105304200
Registrant FAX:+1.3105301652
Registrant Email:jwgranzow@gmail.com
Admin ID:P-1901109
Admin Name:Jay Granzow
Admin Organization:Jay Granzow
Admin Street1:22930 Crenshaw Blvd. Suite D
Admin City:Torrance
Admin State/Province:CA
Admin Postal Code:90505
Admin Country:US
Admin Phone:+1.3105304200
Admin FAX:+1.3105301652
Admin Email:jwgranzow@gmail.com
Tech ID:P-9055753
Tech Name:Jay Granzow
Tech Organization:Jay Granzow
Tech Street1:22930 Crenshaw Blvd. Suite D
Tech City:Torrance
Tech State/Province:CA
Tech Postal Code:90505
Tech Country:US
Tech Phone:+1.3105304200
Tech FAX:+1.3105301652
Tech Email:jwgranzow@yahoo.com
Billing ID:P-1901109
Billing Name:Jay Granzow
Billing Organization:Jay Granzow
Billing Street1:22930 Crenshaw Blvd. Suite D
Billing City:Torrance
Billing State/Province:CA
Billing Postal Code:90505
Billing Country:US
Billing Phone:+1.3105304200
Billing FAX:+1.3105301652
Billing Email:jwgranzow@gmail.com
Sponsoring Registrar ID:7091-KS
Sponsoring Registrar Organization:Key Systems GmbH
Sponsoring Registrar Street1:Im Oberen Werk 1
Sponsoring Registrar City:St. Ingbert
Sponsoring Registrar Postal Code:66386
Sponsoring Registrar Country:DE
Sponsoring Registrar Phone:+49.68949396850
Sponsoring Registrar FAX:+49 (0)68949396851
Sponsoring Registrar Website:http://www.key-systems.net/
Name Server:NS2.SLCIT.COM
Name Server:NS1.SLCIT.COM
DNSSEC:Unsigned


