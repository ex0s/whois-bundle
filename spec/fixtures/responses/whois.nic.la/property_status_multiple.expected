#status
  %s == ["TRANSFER PROHIBITED", "RENEW PERIOD"]
